src/components/articles/Article.tsx

```diff
@@ -1,9 +1,10 @@
 import IArticleProps from '../../lib/interfaces/IArticleProps';
 import { Wrapper } from './Article.styled';
+import { isMobile } from 'react-device-detect';
 
 const Article = ({ children, margin }: IArticleProps) => {
     return (
-        <Wrapper primary={margin}>
+        <Wrapper primary={margin} isMobile={isMobile}>
```

Comment:
I'd say, there is no need for this parameter, and can be easily adjusted using media queries in Article.styled.tsx